include/ceed/ceed.h

```diff
@@ -494,22 +494,36 @@ CEED_EXTERN const char *const CeedQuadModes[];
 /// @ingroup CeedBasis
 typedef enum {
   /// Line
-  CEED_LINE = 1 << 16 | 0,
+  CEED_FE_TOPO_LINE = 1 << 16 | 0,
   /// Triangle - 2D shape
-  CEED_TRIANGLE = 2 << 16 | 1,
+  CEED_FE_TOPO_TRIANGLE = 2 << 16 | 1,
   /// Quadralateral - 2D shape
-  CEED_QUAD = 2 << 16 | 2,
+  CEED_FE_TOPO_QUAD = 2 << 16 | 2,
   /// Tetrahedron - 3D shape
-  CEED_TET = 3 << 16 | 3,
+  CEED_FE_TOPO_TET = 3 << 16 | 3,
   /// Pyramid - 3D shape
-  CEED_PYRAMID = 3 << 16 | 4,
+  CEED_FE_TOPO_PYRAMID = 3 << 16 | 4,
   /// Prism - 3D shape
-  CEED_PRISM = 3 << 16 | 5,
+  CEED_FE_TOPO_PRISM = 3 << 16 | 5,
   /// Hexehedron - 3D shape
-  CEED_HEX = 3 << 16 | 6,
+  CEED_FE_TOPO_HEX = 3 << 16 | 6,
 } CeedElemTopology;
```

Comment:
I'd rather not have `FE` here. libCEED is for *element-based* discretization, but not strictly *finite element* discretization. We don't have examples, but I think we'd like to support methods like spectral volume and spectral difference. We can say `CEED_ELEM_HEX` or `CEED_TOPOLOGY_HEX` or `CEED_ELEM_TOPOLOGY_HEX`.

Reply:
Fair point here. I think CEED_TOPOLOGY_HEX makes more sense and is about the same length as the current

Reply:
Right. I'll update that.